<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTranslate</class>
 <widget class="QWidget" name="frmTranslate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>615</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Translate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit{
	background-color:white;
	padding:15px 15px;
}

QPushButton{
	background-color:rgb(47, 139, 200);
	border-radius:5px;
	color:white;
}</string>
  </property>
  <widget class="QPushButton" name="btnTranslate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbL1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Language</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbL2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Language</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>write your text here!</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmbL1</tabstop>
  <tabstop>btnTranslate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
